nvtabular/column_selector.py

```diff
@@ -40,6 +40,12 @@ def grouped_names(self):
             names.append(tuple(subgroup.names))
         return names
 
+    def __nested_check(self, nests=0):
```

Comment:
I think the convention for a private method is just one underscore:
```suggestion
    def _nested_check(self, nests=0):
```